openhabian-setup.sh

```diff
@@ -132,6 +132,8 @@ else
   load_create_config
   openhabian_console_check
   openhabian_update_check
+  jsscripting_npm_check "openhab"
```

Comment:
wrong branch use the other one ($UNATTENDED is set for  unattended (=initial) installations so this is where your auto-install must go)